### PR TITLE
capstone one project Request

### PR DESCRIPTION
**Pull request by juaryimami**
In this capstone project:
---
- I tried to make a tour agent website for travelers from All over the world to visit Ethiopia.
- In this websites I almost used all the best practices of Html and JavaScript as much as possible
- I also tried to collect best pictures to make my website more attractive.
   Thank you for reviewing my project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ on my design and also animations are applied.
 ## Live Demo
 You can view my hosted website in the link shown below.
 GitHub: [@JUPAC](https://juaryimami.github.io/)
```
Comment (Munsa1) [inline README.md:10]: - [ ] Kindly record a video presenting the features of the project and a pice of code you found interesting.
\
image

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ on my design and also animations are applied.
...
-## :hammer: Built With
+#### :hammer: Built With
```
Comment (Munsa1) [inline README.md:13]: - [ ] It is a requirement tht the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: CSS/mob-style.scss [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+  &__icons {
+    margin: 0.5rem 0;
+    align-items: center;
+    justify-content: space-between;
+    width: 11rem;
+  }
+}
+
+.works {
+  &__cards {
```
Comment (Munsa1) [inline CSS/mob-style.scss:75]: - [ ] The cards are five in total.Kindly add one more card so as to mtch the given design as this is a requirement.
-
Your Design
image

Given design
image

File: CSS/mob-style.scss [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+    grid-template-rows: repeat(3, auto);
+    row-gap: 5.5rem;
+    justify-content: center;
+    padding: 7.125rem 1.5rem;
+  }
+
+  &__card {
```
Comment (Munsa1) [inline CSS/mob-style.scss:82]: - [x] There is a section missing in your design, this is the partners section in the index page. Kindly add it.

Your design
image

Given design
image

- Note to use flexbox to align the items in this section.

File: CSS/mob-style.scss [hunk 5]
```diff
@@ -0,0 +1,295 @@
+@use 'variables' as *;
+
+.navbar {
+  padding: 1rem 1.5rem;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+
```
Comment (Munsa1) [inline CSS/mob-style.scss:8]: - [ ] The mobile menu is not working on the about page in mobile view, Kindly fix this issue.

image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @juaryimami, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._
- [ ] Due ti insufficient files changed on your repo, I have given my review on your CSS file.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/mob-style.scss (hunk 5 above)
Comment (juaryimami) [reply CSS/mob-style.scss:8]: Thank you

File: README.md (hunk 1 above)
Comment (juaryimami) [reply README.md:10]: okay I will prepare the video

File: CSS/mob-style.scss (hunk 3 above)
Comment (juaryimami) [reply CSS/mob-style.scss:75]: actually mine also 5

File: index.html [hunk 6]
```diff
@@ -108,7 +108,7 @@ <h3>Quality Photos</h3>
                     <p>You will get professional Photography as part of every package to make sure your best moments are
                         captured beutifully forever.</p>
                 </div>
-                <div class="features__cards">
+                <div class="features__cards fifth__card">
                     <i class="fa fa-users"></i>
                     <h3>World Class Tour Guides</h3>
                     <p>We have the most seasoned and experienced tour guides who know the locations inside out along
```
Comment (ghost) [inline index.html:114]: - [ ] As the previous reviewer suggested, kindly record 5 minutes loom video about your project and share the link. This is an important project documentation practice and also remember that it is a project requirement.
- [ ] Kindly include the `partners` section missing on your page to match the original  design

### Your design

image

### Missing Section

image

- [ ] In your `about` page (mobile version), when you click on the humburger button, your mobile menu is not showing up. Kindly fix it

image

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @juaryimami , 

Good job implementing some of the changes requested.
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ No linter error
✅ README is okay

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (juaryimami) [reply index.html:114]: Kindly sir, my menu works fine when am clicking the hum burger icon of about menu page

Comment (ghost) [reply index.html:114]: > Kindly sir, my menu works fine when am clicking the hum burger icon of about menu page

Kindly request another review with the changes made and see if the issue persist 👍

File: README.md [hunk 7]
```diff
@@ -8,14 +8,18 @@ on my design and also animations are applied.
 ## Live Demo
 You can view my hosted website in the link shown below.
 GitHub: [@JUPAC](https://juaryimami.github.io/)
-## :hammer: Built With
+(https://www.loom.com/share/b9655fa2600b44bea8b4d2c69f842b66)
```
Comment (HAMMAS-SALEEM) [inline README.md:11]: [Optional]

- [ ] Your loom video should be an overview of your application. This is like an introductory video of your application where you introduce your applictaion by telling and showing the viewers your application. Please keep in mind next time brother :blush:

Example of a good introductory video
https://www.loom.com/share/b0c1075c2a8b4d7faf58eaa39edc12b7

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success

## Hey @juaryimami :wave:

Great work on implementing the requested changes by the previous reviewers. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

:heavy_check_mark: Following the GitHub Flow
:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Followed Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by juaryimami** [comment]
Thank you

**Pull request merged**